### PR TITLE
Update examples/spark for v1beta3

### PR DESCRIPTION
**Pull request by bcbroussard**
Small part of https://github.com/GoogleCloudPlatform/kubernetes/issues/6584.

### CODE REVIEW

**Comment by nikhiljindal** [comment]
Thanks a lot for the PR @bcbroussard
v1beta3 is now the default API version. We can just replace the v1beta1 json files by beta3 ones.
Instead of adding a v1beta3 folder, can you just replace the top level beta1 files?

Thanks!

File: examples/spark/v1beta3/spark-master-service.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "metadata": {
+    "name": "spark-master",
+    "labels": {
+      "name": "spark-master"
+    }
+  },
+  "spec": {
+    "ports": [
+      {
+        "name": "",
```
Comment (nikhiljindal) [inline examples/spark/v1beta3/spark-master-service.json]: You can just delete this field

**Comment by bcbroussard** [comment]
@nikhiljindal - Sure thing, I was wondering about that, but decided to play it safe and follow the pattern Tim and others were using. Making the updates on my other PR's too

File: examples/spark/v1beta3/spark-master-service.json (hunk 1 above)
Comment (bcbroussard) [inline examples/spark/v1beta3/spark-master-service.json]: Thanks, missed that one. Updated.

**Comment by bcbroussard** [comment]
@nikhiljindal - commits squashed, should be ready to go

**Comment by nikhiljindal** [comment]
LGTM, Thanks!

**Pull request merged**